### PR TITLE
Bump deCONZ to 2.19.3 (stable)

### PR DESCRIPTION
**Pull request by beginnersguide**
Bump deCONZ to 2.19.3 (stable)

### CODE REVIEW

File: deconz/config.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-version: 6.17.0
+version: 6.18.0
```
Comment (pvizeli) [inline deconz/config.yaml:2]:
```suggestion
version: 6.17.1
```

File: deconz/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 6.18.0
```
Comment (pvizeli) [inline deconz/CHANGELOG.md:3]:
```suggestion
## 6.17.1
```

**Comment by Prior99** [comment]
Is anything still missing for this to be merged?

**Comment by beginnersguide** [comment]
No, not that I know

Am 7. Dezember 2022 13:26:02 schrieb Frederick Gnodtke 
***@***.***>:

**Pull request merged**